solo-chains/runtime/starlight/src/genesis_config_presets.rs

```diff
@@ -253,10 +259,45 @@ fn starlight_testnet_genesis(
         .collect();
 
     let para_ids: Vec<_> = container_chains
+        .clone()
```

Comment:
`.iter().cloned()` is always better than `.clone().into_iter()`, because it iterates over the source vec once instead of twice.

Reply:
fixed